fds.go

```diff
@@ -95,8 +95,12 @@ type fd struct {
 	Addr    string `json:"addr,omitEmpty"`
 }
 
-func (f *fd) associateFile(name string, file *os.File) {
-	f.file = newFile(file.Fd(), name)
+func (f *fd) associateFile(name string, osFile *os.File) {
+	f.file = &file{
+		osFile,
```

Comment:
I'm confused... Why not just use the `*os.File` directly?

Reply:
From the comment over file:

https://github.com/ngrok/tableroll/blob/e1178d6bbd96ac7c5028cfdb3c656359040f001f/fds.go#L54-L60

This bit of code is inherited from the original tableflip. It still applies I think.